manifests/init.pp

```diff
@@ -261,8 +271,19 @@
   }
 
   if $::osfamily == 'RedHat' {
+    if $package_pcs == undef {
+      $_package_pcs = true
```

Comment:
@alvagante I don't think so:

> If the first character is an underscore, that variable should only be accessed from its own local scope; using qualified variable names where any namespace segment begins with _ is deprecated.

src: https://docs.puppetlabs.com/puppet/latest/reference/lang_reserved.html#variables

Also I just used the syntax which is used for others packages as "corosync" or "pacemaker" where you can specify if it's installed and what version.


So if underscores are problem then I expect another commit which will remove them at once.